_templates/makeRule/new_rule/rule.ejs.t

```diff
@@ -0,0 +1,28 @@
+---
+to: app/stacks/cumulus/resources/rules/<%= dataset_name || 'DSNAME' %>/<%= dataset_name || 'DSNAME' %>___<%= version_str || '1' %>_<%= start_date_YYYY %>.json
+---
+{
+  "name": "<%= dataset_name || 'DSNAME' %>___<%= version_str || '1' %>_<%= start_date_YYYY %>",
+  "state": "DISABLED",
+  "rule": {
+    "type": "onetime"
+  },
+  "provider": "maxar",
+  "collection": {
+    "name": "<%= dataset_name || 'DSNAME' %>",
+    "version": "1"
+  },
+  "workflow": "DiscoverAndQueueGranules",
+  "meta": {
+    "discoverOnly": false,
+    "providerPathFormat": "<%= provider_path_format %>",
+    "rule": {
+      "state": "DISABLED"
+    },
+    "startDate": "<%= start_date_YYYY %>-01-01T00:00:00Z",
+    "endDate": "<%= (start_date_YYYY + 1) %>-01-01T00:00:00Z",
+    "step": "P1D"
+  }
+}
+
```

Comment:
I've made a few tweaks here as well:

1. removed default `'DSNAME'` since that is not something we would ever want to use, nor do we want any sort of default
2. removed version default as well
3. corrected path of "to" in the frontmatter
4. inserted version var as value of collection.version property
5. added message to frontmatter so that we can copy-paste the message as a basis for running the generator multiple times, simply changing the value for year each time (this avoids having to go through the prompts multiple times)

```suggestion
---
to: app/stacks/cumulus/resources/rules/<%= collectionName %>/v<%= collectionVersion %>/<%= collectionName %>___<%= collectionVersion %>_<%= year %>.json
message: >
  hygen rule new
  --provider <%= provider %>
  --collection-name <%= collectionName %>
  --collection-version <%= collectionVersion %>
  --provider-path-format <%= providerPathFormat %>
  --year <%= year %>
---
{
  "name": "<%= collectionName %>___<%= collectionVersion %>_<%= year %>",
  "state": "DISABLED",
  "rule": {
    "type": "onetime"
  },
  "provider": "<%= provider %>",
  "collection": {
    "name": "<%= collectionName %>",
    "version": "<%= collectionVersion %>"
  },
  "workflow": "DiscoverAndQueueGranules",
  "meta": {
    "discoverOnly": false,
    "providerPathFormat": "<%= providerPathFormat %>",
    "rule": {
      "state": "DISABLED"
    },
    "startDate": "<%= year %>-01-01T00:00:00Z",
    "endDate": "<%= parseInt(year) + 1 %>-01-01T00:00:00Z",
    "step": "P1D"
  }
}
```